### PR TITLE
added pixel_order

### PR DESCRIPTION
**Pull request by caternuson**
Adds ability to specify different pixel color orders.
#1

### CODE REVIEW

**Comment by ladyada** [comment]
@tannewt other than renaming ORDER to order, any thoughts?

File: neopixel.py [hunk 1]
```diff
@@ -36,6 +36,12 @@
...
 
+# Pixel color order constants
+RGB = (0, 1, 2)
+GRB = (1, 0, 2)
+RGBW = (0, 1, 2, 3)
+GRBW = (1, 0, 2, 3)
+
 class NeoPixel:
     """
     A sequence of neopixels.
```
Comment (kattni) [inline neopixel.py]: The parameters here should be updated to reflect the new addition.

Comment (caternuson) [reply neopixel.py]: Yep. I'll update the docs if this flies. At this point I'm wondering if this general approach is even good.

Comment (sommersoft) [reply neopixel.py]: Didn't even notice this PR was open. I just put in a PR to update the docs to the new format. Suggest waiting until that is merged to update doc info.

**Comment by caternuson** [comment]
There you go. Updated the readme to show usage and also added an example.

**Comment by tannewt** [comment]
Looks good to me. @kattni please take another look and merge if it looks good.

**Pull request merged**